docs/source/blocks/settings.md

```diff
@@ -102,7 +102,22 @@ const customBlocks = {
           }
         ]
       }
-    }
+    },
+     // A block can have elements included in ToC (Table of Contents block)
+    tocEntry: (blockData, data) => {
+    // Arguments of the function will be the data of the block
+    // which will interact with ToC and the data of ToC
+      // If a singe entry is needed, the function will return an array with the
+      // level and the element to be placed in ToC
+      return [level, title]
+      // If there are more entries to be displayed in Toc, the function will
+      // return an object like below:
+      return {
+        // Each nested array represents an elememnt at a specific level
+        // to be displayed in the ToC
+        entries: [[level, title], [level, title]]
+      }
+    },
```

Comment:
I would remove the part about multiple entries, and the related changes in View.jsx. It's not a bad idea, but it's not needed for this block, and I think it's best to wait to add it until there is a concrete use case, so that we can make sure it's needed.